.checkstyle/checkstyle.xml

```diff
@@ -94,7 +94,7 @@
         </module>
         <module name="ClassDataAbstractionCoupling">
             <!-- default is 7 -->
-            <property name="max" value="20"/>
+            <property name="max" value="21"/>
```

Comment:
Do you rememebr for which class did you changed it? Can we use annotation directly on the clss instead as we did with `KafkaAssemblyOperator`?